src/Proxy.h

```diff
@@ -50,6 +50,11 @@ namespace sdbus::internal {
              , std::string destination
              , std::string objectPath );
 
+        const std::string& getObjectPath() const override
+        {
+            return objectPath_;
+        }
```

Comment:
Ditto, defining the method in the `.cpp` file, see comment above...

Reply:
Done